src/plugins/oSnap/plugin.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "oSnap by UMA",
+  "version": "1.0.0",
+  "author": "UMA",
+  "website": "https://safe.global",
+  "icon": "ipfs://QmQjZaheMTLRrk22mrGCkGk2LNoNqqYMR8Bxtwa8mBHyc9",
+  "defaults": {
```

Comment:
If the Osnap plugin doesn't require settings it would be better to add these values elsewhere. As having this here will open the JSON field when selecting the plugin.

Reply:
i see, ok, i guess this is related to your other comment!

Reply:
removing the defaults still allows the plugin to be clicked and modal open to add custom json. that seems unexpected

Reply:
Yes, this is a bug, we should also disable the edit modal when there are no defaults